File: src/main/java/com/prasad/nithin/xmas/entity/Question.java
/**
 * 
 */
package com.prasad.nithin.xmas.entity;

import java.util.List;

import javax.persistence.ElementCollection;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author 982168
 *
 */

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Question {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private String questionId;
	private String questionText;
	private String description;
	
	@ElementCollection
	private List<TEAM> team;
	
}

File: src/main/ng-app/xmas-ui/src/app/datamodel/Question.ts
import { Teams } from "./Teams";

export class Question{
    

    constructor(private _questionId: String,private _questionText: String,private _description : String,private _teams :Teams[]){

    }

    get questionText(){
        return this._questionText;
    }

    get description(){
        return this._description;
    }

    get questionId(){
        return this._questionId;
    }

    
    get teams(){
        return this._teams;
    }

    set questionText(questionText){
        this._questionText=questionText;
    }

    set description(desciption){
        this._description=desciption;
    }

    set questionId(questionId){
        this._questionId=questionId;
    }

    
    set teams(team){
        this._teams=team;;
    }

    getRequestObject() {
        let data = {
            "questionId":this._questionId
        }
        return data;
    }

}